pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -16,6 +16,9 @@ var (
 	controllerRoles = []rbac.ClusterRole{}
 	// controllerRoleBindings is a slice of roles used for controllers
 	controllerRoleBindings = []rbac.ClusterRoleBinding{}
+
+	ReadWrite = []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"}
```

Comment:
@deads2k copied from kube, i guess this will make better experience for people writing the rules? ;-)